StreamChatClient_v3/Controllers/ChannelListController.swift

```diff
@@ -134,13 +139,47 @@ public class ChannelListControllerGeneric<ExtraData: ExtraDataTypes>: Controller
     }
 }
 
+// MARK: - Channel muting
+
+public extension ChannelListControllerGeneric {
+    typealias MuteChannelCompletion = (Error?) -> Void
+
+    /// Mutes the channel with provided **cid**.
+    /// - Parameters:
+    ///   - cid: The channel identifier.
+    ///   - completion: Called when the channel is muted. If the api-call fails, the completion
+    /// is called with an error.
+    func muteChannel(cid: ChannelId, completion: MuteChannelCompletion? = nil) {
+        channelUpdater.muteChannel(cid: cid,
+                                   mute: true,
+                                   completion: completion)
+    }
+
+    /// Unmutes the channel with provided **cid**.
+    /// - Parameters:
+    ///   - cid: The channel identifier.
+    ///   - completion: Called when the channel is unmuted. If the api-call fails, the completion
+    /// is called with an error.
+    func unmuteChannel(cid: ChannelId, completion: MuteChannelCompletion? = nil) {
+        channelUpdater.muteChannel(cid: cid,
+                                   mute: false,
+                                   completion: completion)
+    }
```

Comment:
This also needs to be tested. Again, a simple test that the `channelUpdater` is called with the correct values. Check `test_startUpdating_callsChannelQueryUpdater` for reference.

Reply:
@VojtaStavik done